hello-spring/src/main/resources/openapi/api.yaml

```
@@ -51,3 +51,40 @@ paths:
                       - a
                       - b
                       - c
+  /sum/body:
+    get:
```

oops, meant for this to be a POST.
```suggestion
    post:
```